### ISSUE TITLE
Make it possible to set padding and width to AlertViewController with a custom view

### ISSUE THREAD

**Issue by fhucho** [AntonPoltoratskyi/NativeUI#8]

**Comment by devpolant** [comment]
Hello @fhucho ! Can you please describe issue in more detail?

**Comment by fhucho** [comment]
Hi @devpolant, I'm trying to to create a native-looking dialog with these requirements:

- The dialog should be fully covered with a custom view. This is not possible right now, because there's a padding between the dialog and the custom view inside it.
- Also, I may need to make it slightly wider than the default width, but this is not necessary.

**Comment by devpolant** [comment]
ok, got your idea. Do you have UI design for it? Maybe you can attach screenshot of the required alert if it is possible.

**Comment by fhucho** [comment]
Yeah, sure, it should look similar to this:

This was designed for Android so it doesn't have to look exactly like this. The shadow, background, corner radius and fonts should be adjusted to look native on iOS.

**Comment by fhucho** [comment]
So... what do you think? :)

**Comment by devpolant** [comment]
@fhucho It doesn't look as native alert. The main purpose of this pod is to provide customisable but native appearance at first.

Native alert doesn't have such UI elements like in your design:
- title as image
- close button in top right corner
- action buttons

I think it's possible to make the following updates:
- For your case it is better not just to remove padding for `contentView`, but to swap the whole `AlertView` with custom view in `AlertController`. Because native actions at the bottom won't look as in your design.

As a solution we can try to pass `UIView` object into some convenience initializer instead of `AlertViewModel` -> you will be responsible for the whole interaction with such custom view.

**Comment by fhucho** [comment]
True, it doesn't look native. What I'm trying to do is to make it look more native by adjusting the animation, shadow, background color and corner radius. This design doesn't have any action buttons, it just contains a header and a scollable list of items.

Also, depends on how you define native, it could be argued that any customization outside of what's possible natively is not native.

Regarding the suggested solution - would my custom view have to include a native-looking background and shadow?
